src/main/kotlin/org/rust/ide/inspections/fixes/import/AutoImportFix.kt

```diff
@@ -116,7 +136,14 @@ class AutoImportFix(path: RsPath) : LocalQuickFixOnPsiElement(path), HighPriorit
         fun findApplicableContext(project: Project, path: RsPath): Context? {
             if (TyPrimitive.fromPath(path) != null) return null
             val basePath = getBasePath(path)
-            if (basePath.reference.resolve() != null) return null
+            if (basePath.reference.multiResolve().isNotEmpty()) return null
+
+            // Don't try to import path in use item
+            if (path.ancestorStrict<RsUseSpeck>() != null) {
```

Comment:
This check seems cheaper than multiresolve, so let's do it first?

Reply:
Order is important here because we use `findApplicableContext` in `RsUnresolvedReferenceInspection`
If result of `findApplicableContext` is null => this path shouldn't be highlighted as unresolved
If result is not null then the path should be highlighted even it's in `use` item but without quick fix